src/foam/core/reflow/Console.js

```diff
@@ -32,7 +32,7 @@ foam.CLASS({
     { name: 'value' },
     {
       name: 'treeRowRenderer',
-      value: function(e) { e.add(this.flowName); }
+      value: function(e) { this.flowName == 'flow' ? e.add('Flow') : e.add(this.flowName); }
```

Comment:
```suggestion
      value: function(e) { e.add(this.flowName); }
```
I'm setting it back - I agree with Kevin's remarks - makes sense.

I do wonder what scenario would have a user trying to access the main block? In a script block (cmd) the main block is this.flowParent